packages/toolkit/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -175,36 +179,41 @@ async function getProjectsValidToTransform(mavenJavaProjects: vscode.QuickPickIt
  * As long as the project contains a .java file and a pom.xml file, the project is still considered valid for transformation,
  * and we allow the user to specify the Java version.
  */
-export async function validateOpenProjects(projects: vscode.QuickPickItem[]) {
+export async function validateOpenProjects(projects: vscode.QuickPickItem[], projectOnLoad = false) {
```

Comment:
I don't totally understand the purpose of `projectOnLoad` and is it supposed to always be `false`?

Reply:
Its a default optional param value of false. Its intended for onload we are triggering a validation check for reporting.

Reply:
The validation should be silent and NOT throw normal errors/toast notifications.

Reply:
I get that and maybe I'm missing something, but do you ever set it to `true`?